packages/components/src/components/assets/TrezorLogo/TrezorLogo.tsx

```diff
@@ -31,22 +31,27 @@ export interface TrezorLogoProps extends ImgHTMLAttributes<HTMLImageElement> {
     height?: string | number;
 }
 
-const TrezorLogo = ({ type, width = 'auto', height = 'auto', ...rest }: TrezorLogoProps) => (
-    <SvgWrapper
-        width={typeof width === 'number' ? `${width}px` : width}
-        height={typeof height === 'number' ? `${height}px` : height}
-        {...rest}
-    >
-        <StyledReactSVG
-            src={LOGOS[type.toUpperCase()]}
-            beforeInjection={(svg: SVGElement) => {
-                if (typeof height === 'number') {
-                    svg.setAttribute('height', `${height}px`);
-                }
-            }}
-            loading={() => <span className="loading" />}
-        />
-    </SvgWrapper>
-);
-
-export { TrezorLogo };
+const Loading = () => <span className="loading" />;
+
+export const TrezorLogo = ({ type, width = 'auto', height = 'auto', ...rest }: TrezorLogoProps) => {
+    const theme = useTheme();
+
+    return (
+        <SvgWrapper
+            width={typeof width === 'number' ? `${width}px` : width}
+            height={typeof height === 'number' ? `${height}px` : height}
+            {...rest}
+        >
+            <StyledReactSVG
+                src={LOGOS[type.toUpperCase()]}
+                beforeInjection={(svg: SVGElement) => {
+                    if (typeof height === 'number') {
+                        svg.setAttribute('height', `${height}px`);
+                    }
+                    svg.setAttribute('fill', theme.TYPE_DARK_GREY);
```

Comment:
without this, the darkmode coloring was not working.

Reply:
What about using some non-legacy color?

Reply:
Tell me which one it shall be :pray:

Reply:
What about `iconDefault` or `iconSubdued`?

Reply:
fixed